enterprise/server/cmd/ci_runner/util/ci_runner_util.go

```diff
@@ -0,0 +1,62 @@
+package ci_runner_util
+
+import (
+	"context"
+	_ "embed"
+	"flag"
+	"path/filepath"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/cmd/ci_runner/bundle"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/platform"
+	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
+	"github.com/buildbuddy-io/buildbuddy/server/remote_cache/cachetools"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	bspb "google.golang.org/genproto/googleapis/bytestream"
+)
+
+const RunnerName = "buildbuddy_ci_runner"
+
+var (
+	RecycledCIRunnerMaxWait = flag.Duration("remote_execution.workflows_runner_recycling_max_wait", 3*time.Second, "Max duration that a ci_runner task should wait for a warm runner before running on a potentially cold runner.")
+	CIRunnerDefaultTimeout  = flag.Duration("remote_execution.workflows_default_timeout", 8*time.Hour, "Default timeout applied to all ci runners.")
```

Comment:
nit: we don't support workflows for on-prem apps, so you can rename these flags `s/workflows/ci_runner/`